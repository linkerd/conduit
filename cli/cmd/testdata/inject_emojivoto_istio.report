
hostNetwork: pods do not use host networking...............................[ok]
sidecar: pods do not have a proxy or initContainer already injected........[warn] -- known sidecar detected in deployment/web
supported: at least one resource injected..................................[warn] -- no supported objects found
udp: pod specs do not include UDP ports....................................[ok]

Summary: 0 of 1 YAML document(s) injected

