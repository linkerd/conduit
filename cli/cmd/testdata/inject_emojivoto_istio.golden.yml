Resource with name "web" of kind "deployment" cannot be injected due to the following reasons:

Ã— pod has a sidecar injected already
---
